web/src/components/Selector.tsx

```diff
@@ -84,40 +84,32 @@ const Selector = ({
   verticalLayout,
   closeDropDown,
   disabledItemTooltip
-}: PropsType): ReactElement => {
-  return (
-    <Wrapper vertical={verticalLayout}>
-      {items.map(item => {
-        if (item.href) {
-          return (
-            <Element
-              key={item.code}
-              as={Link}
-              to={item.href}
-              onClick={closeDropDown}
-              $enabled={true}
-              $selected={item.code === activeItemCode}>
-              <BoldSpacer>{item.name}</BoldSpacer>
-              {item.name}
-            </Element>
-          )
-        } else {
-          return (
-            <Element
-              as={Tooltip}
-              key={item.code}
-              text={disabledItemTooltip}
-              flow='up'
-              $enabled={false}
-              $selected={false}>
-              <BoldSpacer>{item.name}</BoldSpacer>
-              {item.name}
-            </Element>
-          )
-        }
-      })}
-    </Wrapper>
-  )
-}
+}: PropsType): ReactElement => (
+  <Wrapper vertical={verticalLayout}>
+    {items.map(item => {
```

Comment:
Well not your business but i would prefer to call a function in the template and not use all this logic within the template :/

Reply:
Agreed, but not in the scope of this PR